vulnerability-rating-taxonomy.json

```diff
@@ -1,1592 +1,2164 @@
+					"name": "Lack of Security Headers",
+					"type": "subcategory",
+					"children": [{
+							"id": "x_frame_options",
+							"name": "X-Frame-Options",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "cache_control_for_a_non_sensitive_page",
+							"name": "Cache-Control for a Non-Sensitive Page",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-524", "CWE-933"],
+								"cvss_base_vector": "AV:N/AC:H/PR:N/UI:N/S:U/C:L/I:N/A:N"
+							}
+						},
+						{
+							"id": "x_xss_protection",
+							"name": "X-XSS-Protection",
+							"type": "variant",
+							"priority": 5
+						},
+						{
+							"id": "strict_transport_security",
+							"name": "Strict-Transport-Security",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "x_content_type_options",
+							"name": "X-Content-Type-Options",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "content_security_policy",
+							"name": "Content-Security-Policy",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "public_key_pins",
+							"name": "Public-Key-Pins",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "x_content_security_policy",
+							"name": "X-Content-Security-Policy",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "x_webkit_csp",
+							"name": "X-Webkit-CSP",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "content_security_policy_report_only",
+							"name": "Content-Security-Policy-Report-Only",
+							"type": "variant",
+							"priority": 5,
+							"mapping": {
+								"cwe": ["CWE-933"],
+								"cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+							}
+						},
+						{
+							"id": "cache_control_for_a_sensitive_page",
+							"name": "Cache-Control for a Sensitive Page",
+							"type": "variant",
+							"priority": 4,
+							"mapping": {
+								"cwe": ["CWE-933", "CWE-524"],
```

Comment:
CWE-524 should go first as a smaller number, this way we can ensure we are able to locate cwe fields with identical values. In this case there is cache_control_for_a_**non**_sensitive_page that has `"cwe": ["CWE-524", "CWE-933"],`